src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -304,30 +315,25 @@ export default function InvestOption({ claim, optionIndex, openModal, closeModal
               </i>
             )}
             {/* Token Approve buton - not shown for ETH */}
-            {!isEtherApproveState && approveState !== ApprovalState.APPROVED && (
+            {!isNative && notApproved && (
               <ButtonConfirmed
                 buttonSize={ButtonSize.SMALL}
                 onClick={handleApprove}
-                disabled={
-                  approving || approveState === ApprovalState.PENDING || approveState !== ApprovalState.NOT_APPROVED
-                }
-                altDisabledStyle={approveState === ApprovalState.PENDING} // show solid button while waiting
+                disabled={isPendingApprove}
+                altDisabledStyle={isPendingApprove} // show solid button while waiting
               >
-                {approving || approveState === ApprovalState.PENDING ? (
-                  <Loader stroke="white" />
-                ) : (
-                  <span>Approve {currencyAmount?.currency?.symbol}</span>
-                )}
+                {isPendingApprove ? <Loader stroke="white" /> : <span>Approve {currencyAmount?.currency?.symbol}</span>}
               </ButtonConfirmed>
             )}
-            {/*
-              // CURRENTLY TEST ONLY
-              approveState === ApprovalState.APPROVED && (
-                <UnderlineButton disabled={approving || isPendingApproval} onClick={handleRevokeApproval}>
-                  Revoke approval {approving || (isPendingApproval && <Loader size="12px" stroke="white" />)}
+            {
+              // CURRENTLY TEST ONLY (not on prod, barn or ens)
+              IS_TESTING_ENV && isApproved && (
+                <UnderlineButton disabled={approving || isPendingRevoke} onClick={handleRevokeApproval}>
+                  {(approving || isPendingRevoke) && <Loader size="12px" stroke="white" />}
+                  {approving || isPendingRevoke ? 'Revoking approval . . .' : 'Revoke approval'}{' '}
```

Comment:
nit: Why did you add those dots like that? we usually use the tree consecutive dots `...`

Looks a bit strange to me:
![image](https://user-images.githubusercontent.com/2352112/151349773-707bee12-2aad-4be2-a70f-ed9bda3aa628.png)